src/transforms/at-range.js

```diff
@@ -862,31 +970,40 @@ export function wrapInlineAtRange(transform, range, inline) {
 
     const startInlines = startBlock.nodes.slice(startIndex + 1)
     const endInlines = endBlock.nodes.slice(0, endIndex + 1)
-    const startNode = inline.merge({ key: uid() })
-    const endNode = inline.merge({ key: uid() })
+    const startNode = inline.regenerateKey()
+    const endNode = inline.regenerateKey()
 
-    transform.insertNodeByKey(startBlock.key, startIndex - 1, startNode)
-    transform.insertNodeByKey(endBlock.key, endIndex, endNode)
+    transform.insertNodeByKey(startBlock.key, startIndex - 1, startNode, { normalize: false })
+    transform.insertNodeByKey(endBlock.key, endIndex, endNode, { normalize: false })
 
     startInlines.forEach((child, i) => {
-      transform.moveNodeByKey(child.key, startNode.key, i)
+      transform.moveNodeByKey(child.key, startNode.key, i, { normalize: false })
     })
 
     endInlines.forEach((child, i) => {
-      transform.moveNodeByKey(child.key, endNode.key, i)
+      transform.moveNodeByKey(child.key, endNode.key, i, { normalize: false })
     })
 
+    if (normalize) {
+      transform = transform
+        .normalizeNodeByKey(startBlock.key)
+        .normalizeNodeByKey(endBlock.key)
+    }
+
     blocks.slice(1, -1).forEach((block) => {
-      const node = inline.merge({ key: uid() })
-      transform.insertNodeByKey(block.key, 0, node)
+      const node = inline.regenerateKey()
+      transform.insertNodeByKey(block.key, 0, node, { normalize: false })
 
       block.nodes.forEach((child, i) => {
-        transform.moveNodeByKey(child.key, node.key, i)
+        transform.moveNodeByKey(child.key, node.key, i, { normalize: false })
       })
+
+      if (normalize) {
+        transform = transform.normalizeNodeByKey(block.key)
```

Comment:
I have removed them :)

I think we kept this syntax out of nostalgia. Deep in our heart, we have always wished that transform had stayed immutable. Keeping this syntax would have made it easier to change it back :)